pipelines/resources/common/pod-template.yaml

```diff
@@ -0,0 +1,4 @@
+dnsConfig:
```

Comment:
Is this file used in the end?

Reply:
yes...reduce so much time with dns in the Tekton pods, even using dnsmasq